styles.css

```diff
@@ -509,7 +509,7 @@ FUN WITH FORMS
     .hsuforum-thread-header,
```

Comment:
You're assigning the width twice for .hsuforum-thread-header. Maybe remove the .hsuforum-thread-header from this rule and add float: left; to the below .hsuforum-thread-header rule.